packages/node_modules/@ciscospark/react-component-badge/src/styles.css

```diff
@@ -1,9 +1,59 @@
 .badge {
-  height: 18px;
+  position: relative;
+  display: flex;
+  width: 35px;
```

Comment:
Just noticed this static width. What happens when the badge contents get wider than 35px that?

Reply:
![img](https://media1.giphy.com/media/oe33xf3B50fsc/giphy.gif?cid=e1bb72ff5a282cc045676670455e160a)